content/install_maintain/agents/installation.md

```diff
@@ -130,24 +130,30 @@ every agent.
    node instance, or form the deployment logs:    
    ![Script in logs]( /images/manager/agent_installation/logs_script.webp )
 1. Login to the agent's VM.
+1. Prepare the agent directory
+   ```shell
+   sudo mkdir -p /opt/cloudify-agent-6.3.0-ga/
+   sudo chown $(whoami) /opt/cloudify-agent-6.3.0-ga/
```

Comment:
Why set it to be owned by the user? Shouldn't it be root?

Reply:
otherwise the script won't work, even with sudo

Reply:
That doesn't sound right- I thought we set that dir to be owned as root when we installed agents. How does the script fail?

If we don't do that then presumably trying to reinstall an agent as a different user will result in fire and sadness.

Reply:
After further discussion, we're going to leave this part of the instructions alone for the moment as it should only have impact if installing multiple agents on the same machine using this approach.